misc/debug_tools/acrn_trace/README.rst

```diff
@@ -161,13 +163,13 @@ data to your Linux system, and run the analysis tool.
 Build and Install
 *****************
 
-The source files for ``acrntrace`` are in the ``tools/acrntrace`` folder,
-and can be built and installed using:
+The source files for ``acrntrace`` are in the ``misc/debug_tools/acrn_trace``
+directory. To build and install ``acrntrace``, run these commands:
 
 .. code-block:: none
 
    # make
    # make install
 
-The processing scripts are in ``tools/acrntrace/scripts`` and need to be
-copied to and run on your Linux system.
+The processing scripts are in ``misc/debug_tools/acrn_trace/scripts`` and need
+to be copied to and run on your development computer.
```

Comment:
@NanlinXie @dbkinder Do acrntrace_format.py and acrnalyze.py need to be run on the development computer? The example steps above imply that only acrnalyze.py needs to run on the development computer.